buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -1,7 +1,7 @@
 import org.gradle.api.JavaVersion
 
 object Versions {
-    const val precompose = "1.3.14"
+    const val precompose = "1.4.0"
```

Comment:
i just made this up, lemme know if i should remove it

Reply:
It's better to leave it as it.